### PR TITLE
Pass login tokens to the vis apps

### LINKED ISSUE #123: Pass token along to vis apps in querystring
Since we're now using tokens for the authentication, we need to pass them along to the vis apps so they can access private data.

### PR DESCRIPTION
**Pull request by JackWilb**
Closes #123

Adds logic to pass the login token from the main client to the vis apps via the querystring. 

I also had to update yarn.lock, since it wasn't updated in #122 after the version was bumped.

### CODE REVIEW

File: src/views/GraphDetail.vue [hunk 1]
```diff
@@ -93,7 +93,7 @@
                 v-for="app in apps"
                 :key="app.name"
                 class="pl-2"
-                :href="`${app.url}/?workspace=${workspace}&graph=${graph}`"
+                :href="`${app.url}/?workspace=${workspace}&graph=${graph}&token=${userToken}`"
```
Comment (AlmightyYakob) [inline src/views/GraphDetail.vue:96]: I'd prefer if this was `loginToken`, so it's consistent with the other use of it.

```suggestion
                :href="`${app.url}/?workspace=${workspace}&graph=${graph}&loginToken=${userToken}`"
```

**Review by AlmightyYakob — CHANGES_REQUESTED** [review]
Just one small suggestion, otherwise looks good

File: src/views/GraphDetail.vue (hunk 1 above)
Comment (AlmightyYakob) [reply src/views/GraphDetail.vue:96]: Also, it might be good to have a computed function for the end `&loginToken=...`, which would return the empty string if the token isn't present. This would prevent always passing `&loginToken=` whenever the user isn't logged in.

File: src/views/GraphDetail.vue [hunk 2]
```diff
@@ -382,6 +383,9 @@ export default Vue.extend({
     drawerIcon(): string {
       return this.panelOpen ? 'expand_more' : 'expand_less';
     },
+    userToken() {
```
Comment (waxlamp) [inline src/views/GraphDetail.vue:386]: Rename this to `loginToken()` to reflect @AlmightyYakob's comments above.

**Review by waxlamp — CHANGES_REQUESTED** [review]
Thanks for taking initiative on this, @JackWilb. I agree with @AlmightyYakob's overall review/comments.

File: src/views/GraphDetail.vue (hunk 2 above)
Comment (JackWilb) [reply src/views/GraphDetail.vue]: See 89f51e5

File: src/views/GraphDetail.vue (hunk 1 above)
Comment (JackWilb) [reply src/views/GraphDetail.vue]: See 89f51e5

File: src/views/GraphDetail.vue [hunk 3]
```diff
@@ -387,7 +387,7 @@ export default Vue.extend({
       const token = getLoginToken();
 
       if (token !== null) {
-        return `&loginToken=${token}`;
+        return `loginToken=${token}`;
```
Comment (waxlamp) [inline src/views/GraphDetail.vue:390]: I think you want to keep the `#` here; in case there is no token, line 96 above will have a `#` in the URL with nothing following it. Cleaner would be to keep the `#` here so that it won't appear in the URL at all if there's no token.

File: yarn.lock [hunk 4]
```diff
@@ -5852,10 +5852,10 @@ multimatch@^2.1.0:
...
     minimatch "^3.0.0"
 
-multinet@0.16.0:
-  version "0.16.0"
-  resolved "https://registry.yarnpkg.com/multinet/-/multinet-0.16.0.tgz#7361fc87fbc84745f82f45bb8dd5ccd4d6a1f39b"
-  integrity sha512-aqbuRPG5XwGgK1ChhQwGmKkMJcrefbB7H0bKtLIU3OJp1iDwKSWhQ+wdfzPHvPweRt4f798suXdybN1v/uf8hw==
+multinet@0.17.0:
+  version "0.17.0"
+  resolved "https://registry.yarnpkg.com/multinet/-/multinet-0.17.0.tgz#05b30b85d29af8cb877bed502d8cacdabe8796ed"
+  integrity sha512-EC+ALY7ciwSmbkC49NE5A0OmFYxjt/QrTPQf7sXDeL6FUi74UKaammlN3E11T4Pur3kNx686AycmKSbgCkPHNA==
```
Comment (waxlamp) [inline yarn.lock:5858]: A reminder: in general, please keep unrelated changes in their own PRs (even if it's a tiny change like this one).

**Review by waxlamp — CHANGES_REQUESTED** [review]
Looks good overall. Just the one request.

File: src/views/GraphDetail.vue (hunk 3 above)
Comment (JackWilb) [reply src/views/GraphDetail.vue:390]: I had moved it to the `url` param, above in this file, so that if we add another parameter to the fragment it would work as intended. Happy to make this change if you prefer

Comment (AlmightyYakob) [reply src/views/GraphDetail.vue:390]: Same concern that Roni raised. If we're concerned about supporting multiple fragments in the future, it'd probably be good to replace this computed function with a `fragment` computed function, which could aggregate any desired fragment sections, and return something like:

```
// Create this list somehow
const fragments = ['loginToken=abcd', ...];
return fragments.length ? `#${fragments.join("&")}` : '';
```

However, it's not something that really needs to be addressed in this PR, just a thought.

File: yarn.lock (hunk 4 above)
Comment (JackWilb) [reply yarn.lock:5858]: The version is bumped in package.json in master, but not in yarn.lock. When I ran `yarn install`, it was automatically upgraded. Also, version 0.17.0 is required for all the new auth changes. Would you like me to extract this out into its own PR that this Pr depends on?

Comment (AlmightyYakob) [reply yarn.lock:5858]: In that case it looks like we already messed up. I don't think this is worth extracting into its own PR, since this should already be in master anyway.

File: src/views/GraphDetail.vue (hunk 3 above)
Comment (JackWilb) [reply src/views/GraphDetail.vue]: Addressed in 5877967

File: yarn.lock (hunk 4 above)
Comment (JackWilb) [reply yarn.lock:5858]: I'll leave it as is for now and in future make similar changes as a separate PR

**Review by AlmightyYakob — APPROVED** [review]

**Review by waxlamp — APPROVED** [review]
